src/chocolatey/infrastructure.app/rules/VersionMetadataRule.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright © 2023-Present Chocolatey Software, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace chocolatey.infrastructure.app.rules
+{
+    using System.Collections.Generic;
+    using chocolatey.infrastructure.rules;
+    using NuGet.Packaging;
+    using NuGet.Versioning;
+
+    internal sealed class VersionMetadataRule : MetadataRuleBase
+    {
+        public override IEnumerable<RuleResult> validate(NuspecReader reader)
+        {
+            var version = get_element_value(reader, "version");
+
+            // We need to check for the $version$ substitution value, as it in not replaced

```

Comment:
```suggestion
            // We need to check for the $version$ substitution value, as it is not replaced

```

Not sure if this suggestion is correct.  Just wanted to point out that this doesn't read correctly